docs/backends/app/backend_info_app.py

```diff
@@ -21,10 +22,67 @@
 ibis.options.verbose_log = lambda sql: sql_queries.append(sql)
 
 
+def get_single_query_param(key: str, default=None):
+    query_param_list = st.experimental_get_query_params().get(key)
+    if query_param_list:
+        return query_param_list[0]
+    return default
+
+
+def github_paginated_request(url):
+    next_url = url
+    resources = []
+    headers = {
+        'X-GitHub-Api-Version': '2022-11-28',
+        'Accept': 'application/vnd.github+json',
+    }
+    # On Streamlit Cloud, token should be stored as secret.
+    # See:
+    # https://docs.streamlit.io/streamlit-community-cloud/get-started/deploy-an-app/connect-to-data-sources/secrets-management
+    github_token = os.environ.get('GITHUB_TOKEN')
+    if github_token:
+        headers['Authorization'] = f'Bearer {github_token}'
+    with requests.Session() as session:
+        while next_url is not None:
+            resp = session.get(next_url, headers=headers)
+            resp.raise_for_status()
+            resources.extend(resp.json())
+            next_url = resp.links.get('next', {}).get('url')
+    return resources
+
+
+@st.experimental_memo(ttl=ONE_HOUR_IN_SECONDS)
+def ibis_versions_with_raw_data_list():
+    releases = github_paginated_request(
+        'https://api.github.com/repos/ibis-project/ibis/releases'
+    )
+    versions = (release['name'] for release in releases)
+    versions = (
```

Comment:
Might as well make this into a list comprehension since you're immediately turning this into a list in the next line.